core/log/configurator.cpp

```diff
@@ -66,6 +66,8 @@ namespace kagome::log {
               - name: kagome_protocols
           - name: changes_trie
           - name: storage
+            children:
+              - name: trie
```

Comment:
Is it reasonable to make another group instead reusing **changes_trie** group?

Reply:
these are separate structures